resources/views/platform/components/game/screenshots.blade.php

```diff
@@ -39,7 +39,12 @@ function carousel() {
 </div>
 
 <!-- SM+ -->
-<div class="hidden sm:flex mb-3 mx-0 justify-around w-full gap-y-1 gap-x-5">
+<div class="
+    hidden sm:flex justify-around items-center mb-3 px-6 pt-2 pb-2.5 w-full
+    bg-embed border border-embed-highlight gap-y-1 gap-x-5
+    xl:px-4 xl:pt-4 xl:pb-5 xl:rounded-lg xl:mx-0 xl:w-full xl:min-h-[180px]
+"
+>
```

Comment:
![Screenshot 2023-09-03 at 9 14 29 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/4808a879-049e-4f65-a320-ab6072aadf5f)

This lets us still visually have a "straight line" down, like:

![Screenshot 2023-09-03 at 9 11 14 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/5ad6865d-61ce-4dbf-a4f0-e4f5123135b2)

Otherwise, the eye feels a lot of pressure and undue focus towards the strange whitespace around the screenshots.

Reply:
I like the use of "embedded" areas overall!